documentation/modules/security/proc-renewing-ca-certs-manually.adoc

```diff
@@ -17,67 +18,59 @@ Instead, follow the procedure for xref:renewing-your-own-ca-certificates-{contex
 
 .Prerequisites
 
-* The Cluster Operator is running.
+* xref:deploying-cluster-operator-str[The Cluster Operator must be deployed.]
 * A Kafka cluster in which CA certificates and private keys are installed.
+* The OpenSSL TLS management tool to check the period of validity for CA certificates.
+
+In this procedure, we use a Kafka cluster named `my-cluster` within the `my-project` namespace.
 
 .Procedure
 
-. Apply the `strimzi.io/force-renew` annotation to the `Secret` that contains the CA certificate that you want to renew.
+. Apply the `strimzi.io/force-renew` annotation to the secret that contains the CA certificate that you want to renew.
 +
-.Annotation for the Secret that forces renewal of certificates
-[cols="3*",options="header",stripes="none",separator=¦]
-|===
-
-¦Certificate
-¦Secret
-¦Annotate command
-
-¦Cluster CA
-¦_KAFKA-CLUSTER-NAME_-cluster-ca-cert
-m¦kubectl annotate secret _KAFKA-CLUSTER-NAME_-cluster-ca-cert strimzi.io/force-renew=true
-
-¦Clients CA
-¦_KAFKA-CLUSTER-NAME_-clients-ca-cert
-m¦kubectl annotate secret _KAFKA-CLUSTER-NAME_-clients-ca-cert strimzi.io/force-renew=true
+.Renewing the Cluster CA secret
+[source,shell]
+----
+kubectl annotate secret my-cluster-cluster-ca-cert -n my-project strimzi.io/force-renew=true
 
-|===
-+
-At the next reconciliation the Cluster Operator will generate a new CA certificate for the `Secret` that you annotated.
-If maintenance time windows are configured, the Cluster Operator will generate the new CA certificate at the first reconciliation within the next maintenance time window.
+----
 +
-Client applications must reload the cluster and clients CA certificates that were renewed by the Cluster Operator.
+.Renewing the Clients CA secret
+[source,shell]
+----
+kubectl annotate secret my-cluster-clients-ca-cert -n my-project strimzi.io/force-renew=true
+----
 
-. Check the period the CA certificate is valid:
-+
-For example, using an `openssl` command:
+. At the next reconciliation, the Cluster Operator generates new certificates. 
 +
-[source,shell,subs="+quotes"]
-kubectl get secret _CA-CERTIFICATE-SECRET_ -o 'jsonpath={.data._CA-CERTIFICATE_}' | base64 -d | openssl x509 -subject -issuer -startdate -enddate -noout
-+
-_CA-CERTIFICATE-SECRET_ is the name of the `Secret`, which is `_KAFKA-CLUSTER-NAME_-cluster-ca-cert` for the cluster CA certificate and `_KAFKA-CLUSTER-NAME_-clients-ca-cert` for the clients CA certificate.
-+
-_CA-CERTIFICATE_ is the name of the CA certificate, such as `jsonpath={.data.ca\.crt}`.
-+
-The command returns a `notBefore` and `notAfter` date, which is the validity period for the CA certificate.
+If maintenance time windows are configured, the Cluster Operator generates the new CA certificate at the first reconciliation within the next maintenance time window.
```

Comment:
Does it really? I would expect that to not be the case when annotation triggers it. But would need to check the code. @ppatierno do you know it by any chance out of your head?

Reply:
we have this in the procedures for renewing CA certs manually, renewing your own CA certificates, replacing private keys used by the CA certificates generated by the Cluster Operator, and  renewing or replacing CA certificates and private keys with your own

Reply:
Of course, I had to check the code :-)
Even when using the annotation to force renew, this is what we have:

```java
if (maintenanceWindowSatisfied) {
    renewalType = RenewalType.RENEW_CERT;
} else {
    renewalType = RenewalType.POSTPONED;
}
``` 

so it's "postponed" if the maintainance window is not satisfied, so the statement here is right.

Reply:
Great. Thanks for checking @ppatierno